backend/manager/serializers.py

```diff
@@ -22,6 +23,14 @@ class Meta:
         model = Crawler
         exclude = ('cancel_flag',)
 
+    def validate(self, data):
+        request = self.context.get('request')
+        if request.method == 'POST':
+            if Crawler.objects.filter(status__contains='creating').exists() or Crawler.objects.filter(status__contains='running').exists():
```

Comment:
I'm sorry to say it, but I think it would be better to replace `status__contains` to `status_startswith` because there is a little chance that some exception message contains 'creating' or 'running'.

Reply:
Sure. No problem.